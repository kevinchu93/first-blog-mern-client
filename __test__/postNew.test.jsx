import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PostNew from '../src/app/postNew';

test('PostNew component should render as expected', () => {
  expect(toJson(shallow(<PostNew />))).toMatchSnapshot();
});

