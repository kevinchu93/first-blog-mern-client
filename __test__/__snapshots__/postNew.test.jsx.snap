// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostNew component should render as expected 1`] = `
<Route
  render={[Function]}
/>
`;
